index.html

```diff
@@ -4671,140 +4671,168 @@ <h2>Verification Method Rotation</h2>
-cryptographically verifiable statement cannot be reliably determined,
-then the only safe interpretation of revocation is to make it
-apply both forward and backward in time. DID ecosystems that
-take this approach essentially provide cryptographically verifiable
-statements as ephemeral tokens that can be invalidated at any time by
-the DID controller.
-      </p>
-    </section>
+could discover a revoked key and use it to make cryptographically verifiable
+statements with a simulated date in the past.
+        </p>
+        <p>
+Some <a>DID methods</a> only allow the retrieval of the current state of a
+<a>DID</a>. When this is true, or when the state of a <a>DID</a> at the time of
+cryptographically verifiable statement cannot be reliably determined, then the
+only safe interpretation of revocation is to make it apply both forward and
+backward in time. <a>DID</a> ecosystems that take this approach essentially
+provide cryptographically verifiable statements as ephemeral tokens that can be
+invalidated at any time by the <a>DID controller</a>.
+        </p>
+      </section>
 
+      <section class="notoc">
+        <h3>Revocation in Trustless Systems</h3>
 
-    <div class="note">
-      <p>
+        <p>
 Trustless systems are those where all trust is derived from cryptographically
 provable assertions, and more specifically, where no metadata outside of the
 cryptographic system is factored into the determination of trust in the system.
-To verify a signature of proof for a verification method which has been revoked
-in a trustless system, a DID method needs to support either or both of the
-`versionId` or `versionTime`, as well as both the `updated` and `nextUpdate`,
-DID document metadata properties. A verifier can validate a signature or proof
-of a revoked key if and only if all of the following are true:
-      </p>
-      <ul>
-        <li>
-The proof or signature includes the `versionId` or `versionTime` of the DID
-document that was used at the point the signature or proof was created.
-        </li>
-        <li>
-The verifier can determine the point in time at which the signature or proof
-was made, e.g., it was anchored on a blockchain.
-        </li>
-        <li>
-For the resolved DID document metadata, the `updated` timestamp is before, and
-the `nextUpdate` timestamp is after, the point in time at which the signature or
-proof was made.
-        </li>
-      </ul>
-    </div>
+To verify a signature of proof for a <a>verification method</a> which has been
+revoked in a trustless system, a <a>DID method</a> needs to support either or
+both of the `versionId` or `versionTime`, as well as both the `updated` and
+`nextUpdate`, <a>DID document</a> metadata properties. A verifier can validate a
+signature or proof of a revoked key if and only if all of the following are
+true:
+        </p>
+
+        <ul>
+          <li>
+The proof or signature includes the `versionId` or `versionTime` of the  <a>DID
```

Comment:
I agree with this content in its narrow sense, but I don't like how it gives the impression (by focusing only on trustless systems) that the only way to achieve trust is to provide all these properties as part of the cryptographic inputs. The date that a mortgage was signed may not be part of the cryptographic inputs to a verifiable statement, and yet it may be possible to achieve full trust, by deciding that metadata about the event from some other source (e.g., banking records, independent witnesses) is also reliable. In fact, this is likely to happen in a courtroom in many scenarios: metadata from many sources will be evaluated and accepted/rejected. 

My suggestion is to add an extra paragraph right after the &lt;/ul&gt; that says something like: "In systems that are willing to admit metadata other than those constituting cryptographic input, similar trust may be achieved -- but always on the same basis (making a careful judgment about whether a DID document's content at the moment of a signing event authorized the verification method)."

Reply:
I added a minor variation of your language to the specification.